client/issue-lib.js

```diff
@@ -1,6 +1,7 @@
 (function() {
+'use strict';
 
-if (window.Issue && window.IssueList) {
+if (window.Issue) {
   return;
 }
 
```

Comment:
Nit: Since you're modifying this file, can you update the _reviewLevelMetadata query fields to use the casing now used in Monorail (Update-Daily, etc.)?